src/main/java/org/example/game2048/movements/Movement.java

```diff
@@ -0,0 +1,172 @@
+package org.example.game2048.movements;
+import org.example.game2048.point.Point;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Movement {
+
+    private static List<Point> TEMPORARY_LIST;
```

Comment:
to nie jest temporary skoro jest stanem klasy i jeszcze zwracasz to w funkcjach :D

Reply:
Got it!